x-pack/plugins/security_solution/public/management/services/exceptions_list/exceptions_list_api_client.ts

```diff
@@ -43,9 +43,11 @@ export class ExceptionsListApiClient {
       T extends CreateExceptionListItemSchema | UpdateExceptionListItemSchema
     >(
       item: T
-    ) => T
+    ) => T,
+    public readonly version?: string
   ) {
     this.ensureListExists = this.createExceptionList();
+    this.version = version ?? '2023-10-31';
```

Comment:
This date should probably be saved somewhere as a `const` and imported in all the places it is used.